google/cloud/security/common/gcp_type/forwarding_rule.py

```diff
@@ -17,13 +17,14 @@
 See: https://cloud.google.com/compute/docs/reference/latest/forwardingRules
 """
 
-# TODO: The next editor must remove this disable and correct issues.
-# pylint: disable=missing-param-doc
-
+import json
 
 # pylint: disable=too-many-instance-attributes
 class ForwardingRule(object):
-    """Represents ForwardRule resource."""
+    """Represents ForwardRule resource.
```

Comment:
nit: class docstring is good as one line, could you put the Args into the `__init__`?